chromium_src/content/browser/attribution_reporting/attribution_storage_delegate_impl.h

```diff
@@ -1,18 +0,0 @@
-/* Copyright (c) 2023 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at https://mozilla.org/MPL/2.0/. */
-
-#ifndef BRAVE_CHROMIUM_SRC_CONTENT_BROWSER_ATTRIBUTION_REPORTING_ATTRIBUTION_STORAGE_DELEGATE_IMPL_H_
-#define BRAVE_CHROMIUM_SRC_CONTENT_BROWSER_ATTRIBUTION_REPORTING_ATTRIBUTION_STORAGE_DELEGATE_IMPL_H_
-
-#include "content/browser/attribution_reporting/attribution_storage_delegate.h"
-
-#define GetNullAggregatableReports(...)                       \
```

Comment:
same as https://github.com/brave/brave-core/pull/23829/commits/6a6942f46a971a3dc5b1462f4a700816b28c5853#r1648092103 
overriding all of these low level methods doesn't really make sense, we want to hit the entry points and make them no-ops

Reply:
Created a follow-up issue to address this here https://github.com/brave/brave-browser/issues/39223